apps/web/lib/emails/templates/organizer-cancelled-email.ts

```diff
@@ -48,6 +48,18 @@ export default class OrganizerCancelledEmail extends OrganizerScheduledEmail {
       text: this.getTextBody(),
     };
   }
+  protected getDescription(): string {
+    if (!this.calEvent.description) return "";
+    return `
+    <p style="height: 6px"></p>
+    <div style="line-height: 6px;">
+      <p style="color: #494949;">${this.calEvent.organizer.language.translate("description")}</p>
+      <p style="color: #494949; font-weight: 400; line-height: 24px; white-space: pre-wrap;">${
+        this.calEvent.description
+      }</p>
+    </div>
+    `;
+  }
```

Comment:
Can't we just inherit this from `attendee-scheduled-email` ?

Reply:
Yep... I didnt realise we were inheriting them :( Ill get that fixed for the rest of the files too